docs/blog/2018-03-07-why-we-created-the-plugin-library/index.md

```diff
@@ -0,0 +1,80 @@
+
+
+## Card sorting
+
+We gathered, and are continuing to gather, feedback from Github issues, the Product Hive slack community, and face-to-face usability testing. Also, as part of our plugin library UX research, I wanted to know what keywords people might use to search for Gatsby plugins in order to create filters, categories, and tags that make sense. 
+
+I used card sort software to generate dendrograms, other kinds of -grams, and matrices that show the way 20 different Gatsby users typically organize plugins in their minds (big thank you to all who participated here!). Creating filters, categories, and tags are possible future enhancements based off of these results: 
+* [Card Sort Results 1](https://www.optimalworkshop.com/optimalsort/x87kpp82/0mv7111b/shared-results/f24tt2xcxj7di7303t46hec077unam75)
+* [Card Sort Results 2](https://www.optimalworkshop.com/optimalsort/x87kpp82/bt0c01w0/shared-results/1yo4d868hponypjyk1we3xz53sag7uc7)
+
+
+## Prototypes and their evolutions
+
+Here’s a sampling of screenshots that many community members contributed to and gave feedback about. It’s rewarding to see how much the design has progressed (and will keep progressing).
+
+![Plugin Mockup 1](plugin-mockup-1.png)
+
+![Plugin Mockup 2](plugin-mockup-2.png)
+
+![Plugin Mockup 3](plugin-mockup-3.png)
+
+![Plugin Homepage](plugin-homepage.png)
+
+![Plugin Mockup 4](plugin-mockup-4.png)
+
+![Plugin Mockup 5](plugin-mockup-5.png)
+
+
+## Until next time
+
+The purpose of creating the library goes beyond solving a pain point: it also can help people enjoy the full value of the Gatsby ecosystem. The plugin ecosystem is a huge part of what makes Gatsby awesome because plugins and packages make Gatsby extensible and capable of a huge range of powerful things. I hope the library will make it easier for people to find plugins, using them, contribute to them and create them with the help of the Plugin Authoring page in the docs. Anything we can do to make that cycle faster will help the Gatsby ecosystem develop faster and help people build awesome sites. 
+
+Here are some ways you can help make the Gatsby plugin ecoystem great:
+
+We just merged the current version of the plugin library. You can follow its progress and contribute in [Github](https://github.com/gatsbyjs/gatsby/issues/4394).
```

Comment:
Open each of these points with a strong CTA, ie:

* Share feedback for the plugin library at (https://github.com/gatsbyjs/gatsby/issues/4394).
* Create plugins (or publish ones you've already built!). Take a look at the [Plugin Authoring](https://www.gatsbyjs.org/docs/plugin-authoring/#what-files-does-gatsby-look-for-in-a-plugin) guide if you're interested.
* etc